pkgs/development/python-modules/segyio/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, ninja
+, python
+, scikit-build
+, pytest
+, numpy
+, fetchpatch
+}:
+
+stdenv.mkDerivation rec {
+  pname = "segyio";
+  version = "1.9.9";
+
+  patches = [
+    (fetchpatch {
+        url = "https://patch-diff.githubusercontent.com/raw/equinor/segyio/pull/531.patch";
```

Comment:
Can you link the PR and use the link the patch with the commit hash?

The reason for this is because if you change something in your PR (e.g.: the devs ask you to do something else), the way it currently is will make future builds to fail. With the commit hash this will not happen.

Reply:
Dooone @thiagokokada , thanks a lot for explaining.